client/web-playground/src/demo/settings.ts

```diff
@@ -1,8 +1,11 @@
+
+import path from 'path'
 import { type ProviderSettings } from '@opencodegraph/client'
 
 async function getProviders(): Promise<Record<string, ProviderSettings | boolean>> {
     const providerSettings: Record<string, ProviderSettings | boolean> = {
         '../../../../provider/hello-world/index.ts': true,
+        '../../../../provider/techstack/index.ts': { yaml: path.resolve('examples/stackshare.yml') },
```

Comment:
Can you make this use `import.meta.resolve` instead of `path.resolve`? That will be handled by the bundler (vite) more cleanly.  Then you can remove the node polyfill for `path`.